Desktop/components/JASP/Widgets/RCommanderWindow.qml

```diff
@@ -120,7 +120,7 @@ Window
 						selectByMouse:		true
 						readOnly:			true
 
-						RSyntaxHighlighterQuick
+						JC.RSyntaxHighlighterQuick
```

Comment:
@boutinb you made an alias in RCommanderWindow for JC but you never tested it...

Reply:
Oeps